packages/flutter/test/painting/colors_test.dart

```diff
@@ -440,4 +441,13 @@ void main() {
     final Map<String, Object> json = property.toJsonMap(const DiagnosticsSerializationDelegate());
     expect(json.containsKey('valueProperties'), isFalse);
   });
+
+  test('MaterialColor swatch comparison', () {
+    const Map<int, MaterialColor> sampleMap = <int, MaterialColor>{0: Colors.lightBlue, 1: Colors.deepOrange, 2: Colors.blueGrey};
```

Comment:
Style nit, here and below:
```suggestion
    const Map<int, MaterialColor> sampleMap = <int, MaterialColor>{
      0: Colors.lightBlue,
      1: Colors.deepOrange,
      2: Colors.blueGrey,
    };
```

Reply:
okay :)